CleanSlate/common/modifier_definitions/_documentation.info

```diff
@@ -39,6 +39,6 @@ The name of this token is defined in the offmap itself. Gives monthly offmap cur
 
 == Localisation ==
 The key of the modifier will be used for localisation to show in modifier breakdowns. Example:
-horse_archers_OFFENSIVE_FIRE;Horse Archers Attack;Archers montés – Attaque;Ber. Bogenschützen - Angriff;;Ataque de arqueros montados;;;;;;;;;x
+horse_archers_OFFENSIVE_FIRE;Horse Archers Attack;Archers montÃ©s â€“ Attaque;Ber. BogenschÃ¼tzen - Angriff;;Ataque de arqueros montados;;;;;;;;;x
```

Comment:
montÃ©s â€“ and BogenschÃ¼tzen. Generally all the obviously wrong characters like ©, €, ¼ are likely going to exist a lot more often and should be searched for in all files and similarly Ã, though that will give a lot of correct results as well. You also may want to consider splitting up the huge commit into smaller parts, which might be easier to review individually. At the very least one for txt and another for csv files.